workflows/templates/cilium.migrate.yaml

```diff
@@ -99,7 +99,13 @@ spec:
               - name: scriptContent
                 value: |
                   echo "restarting daemonsets on {{inputs.parameters.targetNcn}}"
-                  /srv/cray/scripts/common/restart-ds.sh {{inputs.parameters.targetNcn}}
+                  NODE={{inputs.parameters.targetNcn}}
+                  kubectl get nodes --no-headers=true | awk '{print $1}' | grep -q "^${NODE}$" || { echo "ERROR: Invalid node name ${NODE}"; exit 1; }
+
+                  while read -r a b; do
+                    echo "Restarting pod $b"
+                    kubectl -n $a delete pod $b
```

Comment:
This would be a little clearer with descriptive variable names, like `ns` and `pod` instead of `a` and `b`

Reply:
Fixed